src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php

```diff
@@ -106,6 +106,14 @@ private static function getPhpUnitCliConfigArgument()
      */
     protected static function getKernelClass()
     {
+        if (isset($_SERVER['KERNEL_CLASS'])) {
+            if (!class_exists($class = $_SERVER['KERNEL_CLASS'])) {
+                throw new \RuntimeException(sprintf('Class "%s" not found or cannot be autoloaded. Please check the KERNEL_CLASS value in your phpunit.xml matches the fully-qualified class name of your Kernel or override the WebTestCase::createKernel() method.', $class));
```

Comment:
I don't like the use of *"Please"* in error messages. I'm proposing changing this:

```
Class "%s" not found or cannot be autoloaded. Please check the KERNEL_CLASS
value in your phpunit.xml matches the fully-qualified class name of your Kernel
or override the WebTestCase::createKernel() method.
```

to this:

```
Class "%s" doesn't exist or cannot be autoloaded. Check that the KERNEL_CLASS
value in phpunit.xml matches the fully-qualified class name of your Kernel
or override the WebTestCase::createKernel() method.
```